webpack.config.js

```diff
@@ -24,9 +24,6 @@ module.exports = merge(baseConfig, extractCssConfig, {
       filename: '../index.html',
     }),
   ],
-  resolve: {
```

Comment:
You remove it because now, we don't use the custom SVG files ?

Reply:
We do, but it's not a typical source module, so it's not needed here.